zerver/lib/bugdown/zulip_help_markdown.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+import re
+import os
+import markdown
+import six
+from typing import Any, Dict, Iterable, List, MutableSequence, Optional, Tuple, Union, Text
+from typing.re import Match
+from xml.etree.cElementTree import Element
+
+class HelpMarkdownExtension(markdown.Extension):
+
+    def extendMarkdown(self, md, md_globals):
+        # type: (markdown.Markdown, Dict[str, Any]) -> None
+        """ Add HelpMarkdownExtension to the Markdown instance. """
+        md.registerExtension(self)
+        md.inlinePatterns.add('icon',
+                              Icon(r'!icon\((?P<icon_name>.*?)\)'),
+                              '>backtick')
```

Comment:
I have doubts about this, the order in which this handler should be inserted. I have picked up '>backtick' from `__init__.py` avatar syntax.